src/commons.sh

```diff
@@ -17,47 +17,42 @@ TARGET="qemu"
 BASHPATH=/bin/bash
```

Comment:
Actually this is an unused variable, I want to make a new PR for remove this and other variables; because of this, I did not move this variable to /etc/kworkflow.config.